src/models/backup.ts

```diff
@@ -132,11 +136,16 @@ class Drive {
               if (xhr.readyState === 4) {
                 if (xhr.status === 401) {
                   localStorage.removeItem('driveRefreshToken');
+                  localStorage.driveRevoked = true;
                   resolve(false);
                 }
                 try {
                   const res = JSON.parse(xhr.responseText);
                   if (res.error) {
+                    if (res.error === 'invalid_grant') {
+                      localStorage.removeItem('driveRefreshToken');
+                      localStorage.driveRevoked = true;
```

Comment:
do we need to clear token cache here?

Reply:
No, that only applies to getAuthToken (chrome only). Google doesn't give a 401 when the token gets revoked, and it wasn't actually deleteing them properly before.